avaframe/com1DFA/com1DFA.py

```diff
@@ -909,7 +910,7 @@ def initializeParticles(cfg, releaseLine, dem, inputSimLines='', logName='', rel
         particles, hPartArray = particleTools.initialiseParticlesFromFile(cfg, avaDir, releaseLine['file'].stem)
     else:
         # initialize random generator
-        rng = np.random.default_rng(int(cfg['seed']))
+        rng = np.random.default_rng(int(float(cfg['seed'])))
```

Comment:
What is the reason for the float conversion? Do we actually want non-integer seeds?

Reply:
no the issue was if you ask for seeds in the ini file with 0:9:10 so you expect ['0', '1',' 2'...'9'] but you get ['0.0', '1.0'...'9.0']  and the conversion to int did not work. I hat to go first to float and then int, not sure why...